lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -791,7 +791,10 @@ export class UserAgentApplication {
         this.logger.verbose("Set loading state to pending for: " + scope + ":" + expectedState);
         this.cacheStorage.setItem(`${TemporaryCacheKeys.RENEW_STATUS}${Constants.resourceDelimiter}${expectedState}`, Constants.inProgress);
 
-        const iframe = await WindowUtils.loadFrame(urlNavigate, frameName, this.config.system.navigateFrameWait, this.logger);
+        // render the iframe synchronously if app chooses no timeout, else wait for the set timer to expire
+        const iframe: HTMLIFrameElement = this.config.system.navigateFrameWait ?
```

Comment:
Do I read this right: sync version will run if navigateFrameWait is truthy? Don't we want the opposite?

Reply:
Ah yes, good catch.

Reply:
:-D Sorry. Last minute changes, I need to sleep more :)

Reply:
Done. Thanks @AndreiZe